File: javascript/src/arrays/1929-concatenation-of-array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
const getConcatenation = function (nums) {
  let ans = [];

  for (let i = 0; i < 2; i++) {
    for (let j = 0; j < nums.length; j++) {
      ans.push(nums[j]);
    }
  }

  return ans;
};

module.exports = getConcatenation;

File: golang/arrays/1929-concatenation-of-array.go
package arrays

func getConcatenation(nums []int) []int {
	n := len(nums)
	ans := make([]int, n*2)

	for i := 0; i < 2; i++ {
		for j, num := range nums {
			ans[i*n+j] = num
		}
	}

	return ans
}
